packages/driver/src/cy/listeners.js

```diff
@@ -51,6 +52,17 @@ module.exports = {
 
     // set onerror global handler
     contentWindow.onerror = callbacks.onError
+    contentWindow.addEventListener('unhandledrejection', ({ reason }) => {
+      if (reason.stack) {
+        const details = $stackUtils.getSourceDetailsForFirstLine(reason.stack, config('projectRoot'))
```

Comment:
`unhandledrejection` doesn't provide some things like line, column and filename out of the box and thus we need to extract them manually

Reply:
This sounds good